Zilon.Core/Zilon.TextClient/Program.cs

```diff
@@ -61,25 +63,66 @@ static async System.Threading.Tasks.Task Main()
 
             do
             {
+                PrintState(uiState.ActiveActor.Actor);
+                Console.WriteLine("Input command:");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6388157)